recipes/prgenv-gnu/24.7/mc/extra/reframe.yaml

```diff
@@ -3,4 +3,4 @@ default:
   cc: mpicc
   cxx: mpic++
   ftn: mpifort
-  activation: /user-environment/env/default/activate.sh
+  activation: /user-environment/env/develop/activate.sh
```

Comment:
replace this with:
```yaml
activation:
- uenv view develop
```
This will load the view properly (just sourcing the activation script does not give the same results)